.travis.yml

```diff
@@ -10,6 +10,7 @@ python:
 cache: pip
 
 before_install:
+  - sudo add-apt-repository -y ppa:jonathonf/texlive-2016
```

Comment:
I'm using 2016 as well, but I don't think the files use any recent features. *Which* errors do you mean?